### ISSUE TITLE
Data Model update 

### ISSUE THREAD

**Issue by cyork** [jhu-digital-manuscripts/AnIOp#104]
Start conversation between @markpatton and John → produce an initial data model update proposal

**Comment by jabrah** [comment]
In progress

https://docs.google.com/document/d/1jf8F5kXP5RVkW6BJrlzv4xclEgGd36NPfMXm-g9AyK0/edit?usp=sharing

**Comment by jacobwegner** [comment]
@jabrah / @markpatton / JHU team:

@jtauber  and I took a look at [the feedback document above](https://docs.google.com/document/d/1jf8F5kXP5RVkW6BJrlzv4xclEgGd36NPfMXm-g9AyK0/edit?usp=sharing); below are suggested changes we think we could implement to help address the feedback.

- We feel that continuing to rely on the discovery endpoint (http://aniop-atlas-staging.eldarion.com/wa/discovery/?canvas_id=<canvas_id>`) would be best.

- We would continue to deliver _collections_ based on the canvas id for a given folio but adapt as follows:

**1) As suggested, refactor the `translation-aligment` annotation out as `transcription` and `translation` annotations.**

  1a)  transcription

```json
// PROPOSED transcription containing all lines

{
    "@context": "http://www.w3.org/ns/anno.jsonld",
    "type": "Annotation",
    "id": "https://aniop-atlas-staging.eldarion.com/wa/urn:cite2:hmt:msA.v1:12r/transcription/0/",
    "motivation": "transcribing",
    "body": {
        "type": "TextualBody",
        "language": "grc",
        "format": "text/plain",
        "value": "Μῆνιν ἄειδε θεὰ Πηληϊάδεω Ἀχιλῆος\nοὐλομένην· ἡ μυρί' Ἀχαιοῖς ἄλγε' ἔθηκεν·\nπολλὰς δ' ἰφθίμους ψυχὰς Ἄϊδι προΐαψεν\nἡρώων· αὐτοὺς δὲ ἑλώρια τεῦχε κύνεσσιν\nοἰωνοῖσί τε πᾶσι· Διὸς δ' ἐτελείετο βουλή·\nἐξ οὗ δὴ τὰ πρῶτα διαστήτην ἐρίσαντε\nἈτρείδης τε ἄναξ ἀνδρῶν καὶ δῖος Ἀχιλλεύς ·\nτίς τάρ σφωε θεῶν ἔριδι ξυνἕηκε μάχεσθαι·\nΛητοῦς καὶ Διὸς υἱός· ὁ γὰρ βασιλῆϊ χολωθεὶς\nνοῦσον ἀνὰ στρατὸν ὦρσε κακήν· ὀλέκοντο δὲ λαοὶ .\nοὕνεκα τὸν Χρύσην ἠτίμασεν ἀρητῆρα\nἈτρείδης · ὃ γὰρ ἦλθε θοὰς ἐπὶ νῆας Ἀχαιῶν ·\nλυσόμενός τε θύγατρα· φέρον τ' ἀπερείσι' ἄποινα·\nστέμματ' ἔχων ἐν χερσὶν ἑκηβόλου Ἀπόλλωνος\nχρῡσέῳ ἀνὰ σκήπτρῳ· καὶ λίσσετο πάντας Ἀχαιούς ·\nἈτρείδα δὲ μάλιστα δύω κοσμήτορε λαῶν·\nἈτρεῖδαί τε καὶ ἄλλοι ἐϋκνήμιδες Ἀχαιοί·\nὑμῖν μὲν θεοὶ δοῖεν Ὀλύμπια δώματ' ἔχοντες\nἐκπέρσαι Πριάμοιο πόλιν· εὖ δ' οἴκαδ' ϊκέσθαι·\nπαῖδα δ' ἐμοὶ λύσαιτε φίλην· τὰδ' ἄποινα δεχέσθαι .\nἁζόμενοι Διὸς υἱὸν ἑκηβόλον Ἀπόλλωνα ·\nἜνθ' ἄλλοι μὲν πάντες ἐπευφήμησαν Ἀχαιοὶ .\nαἰδεῖσθαί θ' ἱ̈ερῆα καὶ ἀγλαὰ δέχθαι ἄποινα·\nἀλλ' οὐκ Ἀτρείδῃ Ἀγαμέμνονι ἥνδανε θυμῷ·\nἀλλα κακῶς ἀφίει· κρατερὸν δ' ἐπὶ μῦθον ἔτελλε·"
    },
    "target": [
        "urn:cts:greekLit:tlg0012.tlg001.perseus-grc2:1.1-1.25",
        {
            "type": "SpecificResource",
            "source": {
                "id": "https://rosetest.library.jhu.edu/rosademo/iiif/homer/VA/VA012RN-0013/canvas",
                "type": "Canvas"
            }
        }
    ]
}
```



1b) translation

```json
// PROPOSED first translation snippet
{
    "@context": "http://www.w3.org/ns/anno.jsonld",
    "type": "Annotation",
    "id": "https://aniop-atlas-staging.eldarion.com/wa/urn:cite2:hmt:msA.v1:12r/translation/0/",
    "motivation": "translation",
    "body": {
        "type": "TextualBody",
        "language": "en",
        "format": "text/plain",
        "value": "The wrath sing, goddess, of Peleus' son, Achilles, that destructive wrath which brought countless woes upon the Achaeans, and sent forth to Hades many valiant souls of heroes, and made them themselves spoil for dogs and every bird; thus the plan of Zeus came to fulfillment, from the time when 1 first they parted in strife Atreus' son, king of men, and brilliant Achilles."
    },
    "target": [
        "urn:cts:greekLit:tlg0012.tlg001.perseus-grc2:1.1-1.7",
        {
            "type": "SpecificResource",
            "source": {
                "id": "https://rosetest.library.jhu.edu/rosademo/iiif/homer/VA/VA012RN-0013/canvas",
                "type": "Canvas"
            },
            "selector": {
                "type": "FragmentSelector",
                "region": "xywh=percent:6,23,47,15"
            }
        },
        {
            "type": "SpecificResource",
            "source": {
                "id": "https://aniop-atlas-staging.eldarion.com/wa/urn:cite2:hmt:msA.v1:12r/transcription/0/",
                "type": "Annotation"
            },
            "selector": {
                "type": "TextQuoteSelector",
                "exact": "Μῆνιν ἄειδε θεὰ Πηληϊάδεω Ἀχιλῆος\nοὐλομένην· ἡ μυρί' Ἀχαιοῖς ἄλγε' ἔθηκεν·\nπολλὰς δ' ἰφθίμους ψυχὰς Ἄϊδι προΐαψεν\nἡρώων· αὐτοὺς δὲ ἑλώρια τεῦχε κύνεσσιν\nοἰωνοῖσί τε πᾶσι· Διὸς δ' ἐτελείετο βουλή·\nἐξ οὗ δὴ τὰ πρῶτα διαστήτην ἐρίσαντε\nἈτρείδης τε ἄναξ ἀνδρῶν καὶ δῖος Ἀχιλλεύς ·"
            }
        }

    ]
}

```

@jabrah wrote:

> However, the chosen selector should only reference text found on the associated page. If a translation is associated with some block of text on a different page, that block should go in a new target with its own selector.

Given that we're using the canvas id to determine which annotations are applicable, this _should_ be possible; we'd prefer to revisit this in a second pass once 1a and 1b have been implemented.

**2) As suggested, have named entities target the transcription annotation:**

```json
// PROPOSED named entity with transcription target
{
  "id": "https://aniop-atlas-staging.eldarion.com/wa/urn:cite2:hmt:msA.v1:12r/named-entities/0/",
  "type": "Annotation",
  "label": "Named Entity data for Venetus A VA012RN-0013 text \"Πηληϊάδεω\"",
  "creator": "https://scaife-viewer.org/",
  "body": [
    {
      "purpose": "commenting",
      "type": "TextualBody",
      "value": "Achilles",
      "format": "text/plain"
    },
    {
      "purpose": "identifying",
      "source": "https://en.wikipedia.org/wiki/Achilles",
      "format": "text/html"
    }
  ],
  "target": [
    "urn:cts:greekLit:tlg0012.tlg001.perseus-grc2:1.1@Πηληϊάδεω[1]",
    {
      "type": "SpecificResource",
      "partOf": [
        {
          "id": "https://rosetest.library.jhu.edu/rosademo/iiif/homer/VA/manifest",
          "type": "Manifest"
        }
      ],
      "source": {
        "id": "https://rosetest.library.jhu.edu/rosademo/iiif/homer/VA/VA012RN-0013/canvas",
        "type": "Canvas"
      }
    },
    {
        "type": "SpecificResource",
        "source": {
            "id": "https://aniop-atlas-staging.eldarion.com/wa/urn:cite2:hmt:msA.v1:12r/transcription/0/",
            "type": "Annotation"
        },
        "selector": {
            "type": "TextQuoteSelector",
            "exact": "Πηληϊάδεω"
        }
    }
  ]
}
```

**3) As far as the georeference annotation changes, the ATLAS instance is currently not serving up any "place" entities, just the named entities within the 12r folio.**

We like @jabrah's suggestions around purpose / motivation to help clue clients in on how to further consume the linked data from Pleiades.

As far as updating any targets within the georef annotations, the "translation" annotation in `1b` above is based on a different English translation than the text quote selectors in the Georeference annotation (as discussed in https://github.com/jhu-digital-manuscripts/AnIOp/issues/77).

I think modifying the geo reference annotation to "target" the `translation` annotation is likely out of scope for us.  I listed several approaches in https://github.com/jhu-digital-manuscripts/AnIOp/issues/77#issuecomment-620226927.

TLDR: The existing georef annotation probably can't get more specific in its targeting.

... ... ...

Does what we've proposed for `1)` and `2)` make sense?  If so, we feel we can implement those changes in the next week or two.

We're happy to discuss further in Slack or on a call.

**Comment by jacobwegner** [comment]
Just added a comment to https://docs.google.com/document/d/1fA6LOUORFGv34_lghDmtQwX5Qsj1zc1dnxGXSLkCDDA/edit
